scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -459,4 +458,15 @@ protected void selectionChangedHandler(SelectionEvent event) {
         }
     }
 
+    @Override
+    protected void selectionUpdatedHandler(SelectionUpdatedEvent event) {
+        ListGridRecord[] selectedRecords = this.getSelectedRecords();
+        ArrayList<String> selectedTasksIds = new ArrayList<>();
+        for (ListGridRecord selectedRecord : selectedRecords) {
+            Task task = TaskRecord.getTask(selectedRecord);
+            selectedTasksIds.add(task.getJobId() + "_" + task.getId());
+        }
```

Comment:
It would be nice to have a method which creates returns this task.getJobId() + "_" + task.getId()